arangod/Cache/Table.cpp

```diff
@@ -220,15 +220,16 @@ std::uint32_t Table::logSize() const noexcept { return _logSize; }
 
 Table::BucketLocker Table::fetchAndLockBucket(std::uint32_t hash,
                                               std::uint64_t maxTries) {
+  std::uint32_t index = (hash & _mask) >> _shift;
```

Comment:
NIT

uint32_t & uint64_t -> uint64_t
uint64_t >> uint32_t -> uint64_t

so implicit cast?

Reply:
All of these (`hash`, `_mask`, `_shift`) are uint32_t values, so I don't think an implicit cast is actually happening here.